solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedFeatureStore.java

```diff
@@ -88,17 +87,10 @@ public void update(Map<String,Object> map) {
 
     Map<String,Object> params = null;
 
-    if (map.containsKey(CommonLTRParams.MODEL_PARAMS)) {
-      @SuppressWarnings("unchecked")
-      final Map<String,Object> np = (Map<String,Object>) map
-          .get(CommonLTRParams.MODEL_PARAMS);
-      try {
-        params = (Map<String,Object>) CloneUtils.clone(np);
-      } catch (CloneNotSupportedException e) {
-        throw new SolrException(ErrorCode.BAD_REQUEST, e);
-      }
-    }
-
+    final Map<String,Object> paramsMap = 
+        LTRUtils.createParams(map);
+    params = (null == paramsMap) ? params : paramsMap;
```

Comment:
This if check isn't doing anything.  If paramsMap is null, params is also null since that's what you initialized it to.  Could just do params = createParams() directly.